web-ng/src/app/components/layer-list-item/layer-list-item.component.ts

```diff
@@ -46,6 +55,12 @@ export class LayerListItemComponent implements OnInit {
     this.layerPinUrl = this.sanitizer.bypassSecurityTrustUrl(
       getPinImageSource(this.layer?.color)
     );
+    this.routerService.getFeatureId$().subscribe(id => {
+      this.featureId = id;
+    });
+    this.routerService.getProjectId$().subscribe(id => {
```

Comment:
We should add the subscriptions to a composite Subscription as we do in other classes and unsubscribe on ngOnDestroy().